aiidalab_widgets_base/computational_resources.py

```diff
@@ -817,8 +817,17 @@ def observe_memory_per_machine(change):
 
         super().__init__(children, **kwargs)
 
-    def _configure_computer(self, computer: orm.Computer):
-        """Configure the computer"""
+    def _configure_computer(self, computer: orm.Computer, transport: str):
+        if transport == "core.ssh":
+            self._configure_computer_ssh(computer)
+        elif transport == "core.local":
+            self._configure_computer_local(computer)
+        else:
+            msg = f"invalid transport type '{transport}'"
+            raise common.exceptions.ValidationError(msg)
```

Comment:
`ValidationError` is a part of the `common`'s API:
```suggestion
            raise common.ValidationError(msg)
```